### ISSUE TITLE
podman stop: new long list of failure messages

### ISSUE THREAD

**Issue by edsantiago** [projectatomic/libpod#959]
In window 1:
```
# podman run --name foo fedora sleep 60
```
In window 2: `podman stop foo`

After the 10-second timeout, the command in window 1 reliably spits out a long list of warnings:
```
ERRO[0012] `iptables -t filter -D FORWARD -s 10.88.4.158 ! -o 10.88.4.158 -j ACCEPT` failed: iptables: Bad rule (does a matching rule exist in that chain?).
  (exit status 1)
ERRO[0012] Error deleting network: failed to Statfs "/var/run/netns/cni-43218359-d159-2f94-b3f7-80788b2c4660": no such file or directory    
ERRO[0012] Failed to tear down network namespace for container b3421d55a2cef6935e5cc481bf32d14b179693e097167d77307d1015be319ac8: failed to Statfs "/var/run/netns/cni-43218359-d159-2f94-b3f7-80788b2c4660": no such file or directory                                                   
ERRO[0012] Unable to get exit code of container b3421d55a2cef6935e5cc481bf32d14b179693e097167d77307d1015be319ac8: "error updating container b3421d55a2cef6935e5cc481bf32d14b179693e097167d77307d1015be319ac8 state: error closing network namespace for container b3421d55a2cef6935e5cc481bf32d14b179693e097167d77307d1015be319ac8: Failed to unmount namespace /var/run/netns/cni-43218359-d159-2f94-b3f7-80788b2c4660: no such file or directory"
ERRO[0012] `iptables -t filter -D FORWARD -s 10.88.4.158 ! -o 10.88.4.158 -j ACCEPT` failed: iptables: Bad rule (does a matching rule exist in that chain?).
  (exit status 1)
ERRO[0012] Error deleting network: failed to Statfs "/var/run/netns/cni-43218359-d159-2f94-b3f7-80788b2c4660": no such file or directory    
ERRO[0012] Failed to tear down network namespace for container b3421d55a2cef6935e5cc481bf32d14b179693e097167d77307d1015be319ac8: failed to Statfs "/var/run/netns/cni-43218359-d159-2f94-b3f7-80788b2c4660": no such file or directory                                                   
error closing network namespace for container b3421d55a2cef6935e5cc481bf32d14b179693e097167d77307d1015be319ac8: "/var/run/netns/cni-43218359-d159-2f94-b3f7-80788b2c4660" has already been closed
# echo $?
125
```
podman-0.6.3-1.gitcd39b62.fc28 , also fc27

**Comment by mheon** [comment]
Confirmed locally.

It's trying to run `cleanup()` after the container executes, but for some reason it thinks the network ns still exists after cleanup?

Investigating

**Comment by mheon** [comment]
It's definitely a duplicated `cleanup()` in `podman run` - but I'm really confused as to why. `cleanup` should be harmless (will exit early if cleanup is already complete).

**Comment by mheon** [comment]
Wait, I know what this is. `syncContainer()` is trying to clean up & close the network namespace as part of `UpdateContainer()`.

**Comment by mheon** [comment]
I think we can safely make these errors nonfatal, given that the namespace is already gone. Not sure if we can easily get rid of the error logging though.

**Comment by mheon** [comment]
Yeah, I don't think we can get rid of the error logs, they're coming from the CNI network namespace handler, not our code. A fix to make this not a fatal error is on the way though.

**Comment by mheon** [comment]
Opened #962 to address

**Comment by mheon** [comment]
Fixed by #962
